### ISSUE TITLE
Is it possible to output query positions 

### ISSUE THREAD

**Issue by moorembioinfo** [ksahlin/strobemers#7]
Hi, 

In matches.tsv (python StrobeMap) is it possible to output both reference and query match positions.

Thanks and best regards, 

Matt

**Comment by moorembioinfo** [comment]
Or actually is it fine to assume that the end position on the reference will be from the refpos in the outpt +length of match and never -length of match

**Comment by ksahlin** [comment]
Yes, it is always `refpos + match_length`, never `refpos - match_length`. Reverse complement matches will be under a separate query header named `> query_name Reverse`.

However, your question made me realize that I should probably implement an option to output also the query match length, as it may be slightly different to the reference length since NAMs are approximate.

Best,
K

**Comment by moorembioinfo** [comment]
Brilliant, thanks!
